inference-engine/src/vpu/graph_transformer/src/stages/interpolate.cpp

```diff
@@ -0,0 +1,85 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vpu/frontend/frontend.hpp>
+#include <ie_common.h>
+#include <ie_blob.h>
+#include <ngraph/opsets/opset4.hpp>
+#include <vector>
+#include <unordered_set>
+#include <memory>
+#include <set>
+#include <string>
+
+using namespace InferenceEngine;
+
+namespace vpu {
+
+void FrontEnd::parseInterpolate(const Model& model, const ie::CNNLayerPtr& _layer, const DataVector& inputs, const DataVector& outputs) const {
+    VPU_THROW_UNLESS(inputs.size() <= 4 && inputs.size() >= 1,
+                     "Interpolate stage with name {} must have no more than 4 inputs and no less than 1 input, actually provided {} inputs",
+                     _layer->name, inputs.size());
+
+    VPU_THROW_UNLESS(outputs.size() == 1,
+                     "Interpolate stage with name {} must have only 1 output, actually provided {} outputs",
+                     _layer->name, outputs.size());
+
+    using ngraph::opset4::Interpolate;
```

Comment:
Why is it needed?

Reply:
it was an artifact from some old piece of code. Removed!